app/models/target.rb

```diff
@@ -40,7 +40,14 @@ class Target < ApplicationRecord
   scope :not_student, -> { where.not(role: ROLE_STUDENT) }
   scope :team, -> { where(role: ROLE_TEAM) }
   scope :sessions, -> { where.not(session_at: nil) }
-  scope :non_assignment, -> { where.missing(:assignments) }
```

Comment:
This `non_assignment` scope is only used in two places, and in both cases, used to compute `targets_completed` - an outdated idea, since targets are no longer _completed_ - they're merely _read_ - it's assignments that are completed. In a forward-looking UI, these two things would need to be displayed separately - _"targets read"_ (equivalent to a future _"pages read"_), and _"assignments completed"_ figures.

I think it makes sense to spend a little bit of time to upgrade the UIs showing these _target completion_ figures to show _targets read_ (temporarily) and _assignments completed_ figures instead.

I don't see a true use for such a `non_assignment` scope.